collect_app/src/main/java/org/odk/collect/android/tasks/GoogleSheetsAbstractUploader.java

```diff
@@ -98,8 +99,7 @@ protected void uploadInstances(String selection, String[] selectionArgs, String
 
         Cursor c = null;
         try {
-            c = Collect.getInstance().getContentResolver()
-                    .query(InstanceColumns.CONTENT_URI, null, selection, selectionArgs, null);
+            c = new InstancesDao().getInstancesCursor(null, selection, selectionArgs, null);
```

Comment:
I think it would be better to provide methods so that callers never have to pass in null.